Update requirements.txt

PLanB2008 opened the pull request:
Increase Magic Version to 0.4.24 added libmagic which is needed on a clean lmn72 installation on ubuntu 22.04

PLanB2008 committed:
Update requirements.txt

kiarn reviewed (commented):
Thanks for the PR!
We can sure go to the version 0.4.27, but I don't understand the trick with libmagic : I don't any libmagic python package on my installations. Do you eventually mean a deb package ?
I only have these : 

```
dpkg -l | grep libmag
ii  libmagic-mgc                            1:5.41-3 
ii  libmagic1:amd64                         1:5.41-3
```

Commit:
Update api.py

Added lmn_is_installed option

Commit:
Update multischool.py

Only load multischool cofings when .installed trigger is set

PLanB2008 commented:
Well it seems we need this package on one way or another, not quite sure which is the better way.

Another addition I added to this PR:

Dhe loading of samba configs do not work before linuxmuster is prepared. I added a function to lmn_common api to look for the .installed file and added the check to the init function of school manager.
Maybe theres a better solution for this issue... :)

kiarn commented:
> Well it seems we need this package on one way or another

That's what I don't understand : where do we need this package ?
I don't have it at all and I don't any problem with the install process (or did I miss something ? )
`libmagic` is a very old package without update since 2012 and normally `pip` handles all dependencies automatically.

> Dhe loading of samba configs do not work before linuxmuster is prepared. I added a function to lmn_common api to look for the .installed file and added the check to the init function of school manager. Maybe theres a better solution for this issue... :)

Good catch ! Thanks for it.
